dashboard/app/assets/javascripts/client_state.js

```diff
@@ -36,6 +36,22 @@ dashboard.clientState.reset = function() {
   sessionStorage.clear();
 };
 
+dashboard.clientState.queryParams = function (name) {
```

Comment:
This method has a somewhat complicated contract (with the `name` argument being optional, returning an Object if it's omitted, a string if it's not).  Can we add documentation?